### ISSUE TITLE
clock@schorschii: Desklet appears on top of active Remmina window

### ISSUE THREAD

**Issue by transcengopher** [linuxmint/cinnamon-spices-desklets#481]
```
 * Desklet version/Build date: 1.5
 * Cinnamon version: 4.2.4
 * Distribution: Mint 19.2 Cinnamon
 * Graphics hardware : NVIDIA GeForce GTX 660 (GK106)
 * Graphics card driver: nvidia-driver-430.50-0ubuntu0.18.04.2
 ```

**Notify author of desklet**
@schorschii 

**Issue**
When using Remmina's RDP connection, the desklet "pops up" on top of the active viewport window, carving out a small square hole around it, where I can also see the wallpaper in the corners. The carved hole seems to correspond with desktop grid, as the hole is larger than colored overlay displayed when editing desklet settings.

This does not happen when remmina window has another fullscreen window behind it (like fullscreen FireFox window, for example), and does not show reliably (i.e. after some time of minimizing/reopening remmina window, the desklet stops "misbehaving" and doesn't show up on top anymore).
The desklet doesn't seem to steal user input, as clicks are delivered to Remmina viewport. Hole carved by desklet disappears when clicked, but can reappear shortly after. The hole is not visible on screenshots.

**Comment by schorschii** [comment]
Hi @transcengopher ,
I experienced this issue rarely on some notebooks. Because it doesn't appears on my own computers, I wasn't able to search more closely. When I saw it last it affected all desklets, not only my clock specifically. Can you please place some more desklets on your desktop (and remove the clock temporarily) and report the result? I guess that it is a cinnamon issue.

**Comment by transcengopher** [comment]
@schorschii, yes, I've also just now seen it with the default clock (clock@cinnamon.org), meaning it's not exclusive to your desklet.
Appears to happen when desklet has a UI update, i.e. I didn't see anything (yet) when using a photoframe desklet that did no updates.

**Comment by schorschii** [comment]
I didn't noticed this anymore in recent Cinnamon versions. What's you impression, @transcengopher ? I think it can be closed then.
